lib/model/coin_utils.dart

```diff
@@ -188,6 +192,36 @@ String getCoinTypeName(CoinType type) {
   }
 }
 
+bool isParentCoin(CoinType type, String symbol) {
+  switch (type) {
+    case CoinType.utxo:
+    case CoinType.tendermint:
+      return true;
+    case CoinType.erc20:
+      return symbol == 'ETH';
+    case CoinType.bep20:
+      return symbol == 'BNB';
+    case CoinType.avx20:
+      return symbol == 'AVAX';
+    case CoinType.etc:
+      return symbol == 'ETC';
+    case CoinType.ftm20:
+      return symbol == 'FTM';
+    case CoinType.arb20:
+      return symbol == 'ARB' || symbol.startsWith('ARB') ;
```

Comment:
the parent coin of ARB20 tokens is ETH-ARB20: https://github.com/KomodoPlatform/coins/blob/master/coins#L4874-L4898
that's why ARB20 tokens have `"platform": "ETH-ARB20"` set, like https://github.com/KomodoPlatform/coins/blob/master/coins#L6226

Reply:
thanks, fixed in https://github.com/KomodoPlatform/komodo-wallet/pull/2988/commits/b6435f6bb38d583f13e873388ee052a94389f19b